material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -1199,12 +1199,24 @@ input.md-search__input::placeholder {
     color: var(--nav-fg-color);
     font-size: 16px;
   }
+
+  .md-nav--secondary .md-nav__link--active {
+    background: var(--nav-hover-bg-color);
+    border-radius: 6px;
+  }
 }
 
 @media screen and (min-width: 76.25em) {
   .md-nav--secondary .md-nav__link {
     margin-top: 0;
   }
+
+  .md-nav--secondary .md-nav__link--active {
+    margin-left: -0.7em;
+    padding-left: 0.7em;
+    background: var(--nav-hover-bg-color);
+    border-radius: 6px;
```

Comment:
These two lines seem unnecessary to me. Please double-check though

Reply:
Ah yes, I think they are now since you've added styling to the `@media screen and (min-width: 60em) {` block (lines 1203-1206)